x/did/tests/create_diddoc_test.go

```diff
@@ -414,6 +416,44 @@ var _ = Describe("Create DID tests", func() {
 		_, err := setup.CreateDid(msg, signatures)
 		Expect(err.Error()).To(ContainSubstring(fmt.Sprintf("%s: DID Doc exists", alice.Did)))
 	})
+
+	It("Not Valid: DIDDoc Service Routing Keys field", func() {
+		did := GenerateDID(Base58_16bytes)
+		keypair := GenerateKeyPair()
+		keyId := did + "#key-1"
+
+		msg := &types.MsgCreateDidDocPayload{
+			Id:             did,
+			Authentication: []string{keyId},
+			VerificationMethod: []*types.VerificationMethod{
+				{
+					Id:                   keyId,
+					Type:                 types.Ed25519VerificationKey2020{}.Type(),
+					Controller:           did,
+					VerificationMaterial: BuildEd25519VerificationKey2020VerificationMaterial(keypair.Public),
+				},
+			},
+			VersionId: uuid.NewString(),
+			Service: []*types.Service{
+				{
+					Id:              did + "#service-1",
+					Type:            "type-1",
+					ServiceEndpoint: []string{"endpoint-1"},
+					Accept:          []string{"accept-1"},
+					RoutingKeys:     []string{"invalid value"},
+				},
+			},
+		}
+
+		signatures := []SignInput{
+			{
+				VerificationMethodId: keyId,
+				Key:                  keypair.Private,
+			},
+		}
+		_, err := setup.CreateDid(msg, signatures)
+		Expect(err.Error()).To(ContainSubstring("DID namespace validation failed"))
```

Comment:
Why does test expect this error message? Namespace is not required here

Reply:
Done! ✅